src/com/kainos/tdp/employeemanager/Main.java

```diff
@@ -1,7 +1,33 @@
 package com.kainos.tdp.employeemanager;
+import java.util.Scanner;
 
 public class Main {
+	
 	public static void main(String[] args) {
+		runMenu();
+	}
+	
+	private static void runMenu() {
+		int userInput;
+		Scanner sc = new Scanner(System.in);
+		Menu menu = new Menu();
+		boolean validChoice;
 		
+		do {
+			menu.printMenu();
+			userInput = sc.nextInt();
+			validChoice = handleUserChoice(userInput);
+		}while(!validChoice);
+	}
+
+	private static boolean handleUserChoice(int userInput) {
```

Comment:
```suggestion
	private static boolean checkUserChoiceValid(int userInput) {
```
This would seem to be a better name for this method.